index.js

```diff
@@ -361,16 +361,44 @@ Synopsis.prototype.winner = function () {
 
 Synopsis.prototype.winners = function (n, weights) {
   const results = weights || this.topN()
-  let i, point, upper, winners
+  const pinned = []
+  let count = 0
+  let winners
```

Comment:
can just do `const winners = []` and delete https://github.com/brave-intl/bat-publisher/pull/24/files#diff-168726dbe96b3ce427e7fedce31bb0bcR370

Reply:
good catch!